control/control_loop.go

```diff
@@ -289,14 +289,30 @@ func (l *Loop) startBenchmark(loops int) error {
 
 // Stop stops then loop.
 func (l *Loop) Stop() {
-	if l.running {
-		l.running = false
-		l.logger.Debug("closing loop")
-		l.ct.ticker.Stop()
-		close(l.ct.stop)
-		l.cancel()
-		l.activeBackgroundWorkers.Wait()
-	}
+	l.running = false
+	l.logger.Debug("closing loop")
+	l.ct.ticker.Stop()
+	close(l.ct.stop)
+	l.cancel()
+	l.activeBackgroundWorkers.Wait()
+}
+
+// SetStatus sets l.running.
+func (l *Loop) SetStatus(running bool) {
+	l.running = running
+}
```

Comment:
is this currently not used? Would prefer having Pause/Resume or this, but not both

Reply:
oops yes